packaging/iceberg/src/main/scala/com.snowplowanalytics.snowplow.lakes/TableFormatSetupError.scala

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) 2014-present Snowplow Analytics Ltd. All rights reserved.
+ *
+ * This software is made available by Snowplow Analytics, Ltd.,
+ * under the terms of the Snowplow Limited Use License Agreement, Version 1.0
+ * located at https://docs.snowplow.io/limited-use-license-1.0
+ * BY INSTALLING, DOWNLOADING, ACCESSING, USING OR DISTRIBUTING ANY PORTION
+ * OF THE SOFTWARE, YOU AGREE TO THE TERMS OF SUCH LICENSE AGREEMENT.
+ */
+
+package com.snowplowanalytics.snowplow.lakes
+
+import org.apache.iceberg.exceptions.{ForbiddenException => IcebergForbiddenException, NotFoundException => IcebergNotFoundException}
+
+object TableFormatSetupError {
+
+  // Check if given exception is specific to iceberg format
+  def check(t: Throwable): Boolean =
+    t match {
+      case _: IcebergNotFoundException =>
```

Comment:
Have you considered in which package/class we are going to match for Delta setup errors?

If it's going to be in _this_ package, then it's better to call this package something like `packaging/delta-iceberg` instead of `packaging/iceberg`.

On the other hand, if we need another package for Delta.... then I don't know how it's going to work to have two packages both providing a `TableFormatSetupError` object.

Or maybe you haven't thought that far ahead!!  That's fine.  We can tackle that problem when we work on the Delta setup errors.

Reply:
This package is created purely for dependency management. Otherwise, we would have to include Iceberg dependencies to Hudi artifact. It seems like delta-iceberg dependencies are always coming together therefore I don't think we need separate package/class for Delta setup errors. I will change its name to  `packaging/delta-iceberg` as you suggested 👍